about/faq.rst

```diff
@@ -532,26 +532,25 @@ removed at run-time to dynamically add and remove behaviors.
 More information about Godot's design choices can be found in
 `this article <https://godotengine.org/article/why-isnt-godot-ecs-based-game-engine>`__.
 
-Why does Godot not force users to implement DoD (Data-oriented Design)?
+Why does Godot not force users to implement DoD (Data-Oriented Design)?
```

Comment:
It should match the acronym, so either "DoD (Data-oriented Design)" or "DOD (Data-Oriented Design)".

Reply:
Makes sense. A quick search showed DOD was more common than DoD (and is more consistent with OOP etc) so I changed the other ones to DOD. Doesn't seem to be any other references to DoD anywhere else in the docs so that should be it.